UOP1_Project/Assets/Scripts/EditorTools/SwapPrefabs/Editor/SwapPrefabs.cs

```diff
@@ -0,0 +1,584 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+namespace ChopChop.EditorTools.jandd661
+{
+    public class SwapPrefabs : EditorWindow
```

Comment:
So this is another replace tool? Perhaps this file should be in a separate PR so that this one just focuses on addressing the bug in the original tool?

Reply:
I was trying to do that but, Making a new PR, it just updates this one. How do you start a seperate PR? Do I need to remove my fork? I have been looking around how to do this but it remains unclear.